tests/distributions/test_rice.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from rs_distributions.distributions.rice import Rice
+import torch
+
+
+@pytest.mark.parametrize("test_float_broadcasting", [False, True])
+@pytest.mark.parametrize("batch_shape", [3, 10])
+@pytest.mark.parametrize("sample_shape", [(), (10,)])
+def test_rice_execution(test_float_broadcasting, batch_shape, sample_shape):
```

Comment:
we should think of a cheap way to test the correctness of the gradients

Reply:
is finite differences cheap enough?

Reply:
Have to test the error scale, but not for this PR.